src/tools/interop/interop.cpp

```diff
@@ -596,6 +602,15 @@ class InteropConnection {
         }
         return false;
     }
+    bool GetResumptionTicket(const uint8_t*& Ticket, uint32_t& TicketLength) {
+        if (!WaitForTicket() || ResumptionTicket == nullptr) {
+            return false;
+        }
+        Ticket = new uint8_t[ResumptionTicketLength];
```

Comment:
I don't think we need to copy. Just return the pointer and length.

Reply:
Oh, I see, you're freeing the connection before using this. In that case, return the buffer length, and null out the local pointer.